### ISSUE TITLE
Allow to query jobs in the blockers file without searching for a job 

### ISSUE THREAD

**Issue by EliadCohen** [nathan-weinberg/jeeves#54]
My suggestion is that when having multiple jobs across several different components and having a variation in naming, to allow using the full names from the blockers file to query jenkins without utilizing ```job_search_fields: ```

Usecase would be if just a subset of cross-DFG jobs is required for the report.

**Comment by nathan-weinberg** [comment]
@EliadCohen So a run mode where instead of using `job_search_fields`, it uses the job names defined in `blockers.yaml`, correct?

**Comment by EliadCohen** [comment]
> @EliadCohen So a run mode where instead of using `job_search_fields`, it uses the job names defined in `blockers.yaml`, correct?

Yes

**Comment by nathan-weinberg** [comment]
So this behavior has been pseudo-implemented by `remind.py`; what usecase did you see for it? Having as an option for reporting as well?

**Comment by EliadCohen** [comment]
@nathan-weinberg When compiling a report for multiple teams or a selective report for a certain cross-section of versions/jobs

**Comment by nathan-weinberg** [comment]
@agarcia64 Would this help with the cross DFG FFU report?

**Comment by agarcia64** [comment]
@nathan-weinberg Absolutely, this RFE would streamline the process for reporting since it would remove any chance of reporting non FFU taskforce FFU jobs.

**Comment by ciecierski** [comment]
How we are going to track/maintain list of interesting cross-section jobs?  One solution that comes to my mind is creating report based on jenkins views, which are tracked in our internal repo. It should be possible to get job names from a view instead of providing them in config.
